src/neuroconv/basedatainterface.py

```diff
@@ -20,7 +20,10 @@
 class BaseDataInterface(ABC):
     """Abstract class defining the structure of all DataInterfaces."""
 
+    display_name: Union[str, None] = None
     keywords: List[str] = []
+    common_suffixes: List[str] = []
```

Comment:
```suggestion
    associated_suffixes: List[str] = []
```

Reply:
Changing to 'associated_suffixes' since some of these might refer to 'nearby' files, such as headers (example: `.meta` for SpikeGLX, `.xml` for NeuroScope, etc.)